config/options.yml

```diff
@@ -22,3 +22,9 @@ rpm:
   product_url:     https://github.com/ManageIQ/manageiq
 bundler_version:   2.1.4
 npm_registry:
+rpm_repository:
```

Comment:
Should this be under the rpm section?  I ask because this option kind of conflicts with the rpm.repo_name parameter which is for copr.

Reply:
Does it conflict?

FWIW, I still don't think this belongs here, but I wanted to get something going so that we can get Jansa built.  I think it's strange to put the repo in the rpm (the whole in the part)

Reply:
I see what you mean, but I don't see it as whole in the part.  To me building RPMs includes optionally deploying them somewhere.  i.e. this repo has never been about just the building part (though the repo name suggests otherwise 😄 )

Reply:
> Does it conflict?

I guess it doesn't conflict, but it's redundant?